tests/Integration/PDFObjectTest.php

```diff
@@ -226,7 +226,8 @@ public function testGetSectionText()
 EMC
 (ABC) Tj
 
-[ (a)-4.5(b) 6(c)8.8 ( fsdfsdfsdf[ sd) ] TD ', '/TT1 1.5 Tf (BT )Tj '],
+[ (a)-4.5(b) 6(c)8.8 ( fsdfsdfsdf[ sd) ] TD', '/TT1 1.5 Tf (BT )Tj
+q'],
```

Comment:
"Q" and "q" instructions are related to a clip fonctionnality
surrounding a BT/ET block using those instructions allow to handle 2 streams of text construction.
about fonts, this allow a avoid the font to use for each block
this is why text rendered was strange, because the wrong font was used

the particularity of Q/q instructions is they are defined outside of BT/ET block, so the fix I made identify them and copy them into block commands to be handled into the command loop

about PDFObjectTest, the sample contains already a "q" instruction after the BT/ET block so it's copied at the end of commands

I agree with you it should be better to use the file from #387, but it contains real customers informations.
so a new one should be a good thing

Reply:
OK, thank you for the explanation.

> [...] but it contains real customers informations. so a new one should be a good thing

Why is that a bad thing? What do you mean with "new one"?